src/vivarium_gates_nutrition_optimization_child/components/population.py

```diff
@@ -73,6 +76,9 @@ def setup(self, builder: Builder) -> None:
             "age_smoothing_age_bounds": builder.randomness.get_stream(
                 "age_smoothing_age_bounds", initializes_crn_attributes=True
             ),
+            "subnational_selection": builder.randomness.get_stream(
+                "subnational_selection", initializes_crn_attributes=True
```

Comment:
If you have `initializes_crn_attributes=True` then `subnational` ought to be specified as a randomness key column in the model spec. I don't actually think it needs to be a key column though - identifying simulants by entrance_time, age, and maternal_id should be enough.